server/methods/messageSearch.js

```diff
@@ -8,9 +8,10 @@ import { settings } from '../../app/settings';
 import { readSecondaryPreferred } from '../database/readSecondaryPreferred';
 
 Meteor.methods({
-	messageSearch(text, rid, limit) {
+	messageSearch(text, rid, offset, limit) {
```

Comment:
Changing a Meteor Method's parameter order is a breaking change, which should be avoided.